recipes/mongodb_org_repo.rb

```diff
@@ -47,7 +47,7 @@
     keyserver 'hkp://keyserver.ubuntu.com:80'
     key package_version_major >= 3.2 ? 'EA312927' : '7F0CEB10'
   end
-when 'amazon', 'fedora', 'oracle', 'rhel'
+when 'rhel'
```

Comment:
Gah, I was testing with Foodcritic here and forgot to change this line. Foodcritic complains about the way it was originally but behaves normally when 'oracle' is removed because the 'oracle' platform is included when using the 'rhel' platform family since at least Ohai 7.2.0 (https://github.com/chef/ohai/blob/7.2.0/lib/ohai/plugins/linux/platform.rb#L112).